<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Probabilistic Modelling in Genomics 2016 by embatty</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/github-light.css">
    <meta name="viewport" content="width=device-width">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1>Probabilistic Modelling in Genomics 2016</h1>
        <p>Evaluating assembly methods for the highly repetitive genome of the obligate intracellular bacterium Orientia tsutsugamushi</p>

        <p class="view"><a href="https://github.com/embatty/probgen2016">View the Project on GitHub <small>embatty/probgen2016</small></a></p>


        <ul>
          <li><a href="https://github.com/embatty/probgen2016/zipball/master">Download <strong>ZIP File</strong></a></li>
          <li><a href="https://github.com/embatty/probgen2016/tarball/master">Download <strong>TAR Ball</strong></a></li>
          <li><a href="https://github.com/embatty/probgen2016">View On <strong>GitHub</strong></a></li>
        </ul>
      </header>
      <section>
        <h2>
<a id="references" class="anchor" href="#references" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>References</h2>

<p>Cho, N.-H., Kim, H.-R., Lee, J.-H., Kim, S.-Y., Kim, J., Cha, S., Kim, S.-Y., Darby, A.C., Fuxelius, H.-H., Yin, J., et al. (2007). <a href="http://www.pnas.org/content/104/19/7981.long">The Orientia tsutsugamushi genome reveals massive proliferation of conjugative type IV secretion system and host-cell interaction genes.</a> Proc. Natl. Acad. Sci. U. S. A. 104, 7981–7986.</p>

<p>Koren, S., Harhay, G.P., Smith, T.P., Bono, J.L., Harhay, D.M., McVey, S.D., Radune, D., Bergman, N.H., and Phillippy, A.M. (2013). <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-9-r101">Reducing assembly complexity of microbial genomes with single-molecule sequencing.</a> Genome Biol. 14, R101.</p>

<h2>
<a id="tools" class="anchor" href="#tools" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Tools</h2>

<p><strong>Canu</strong></p>

<p>Koren S, Walenz BP, Berlin K, Miller JR, Phillippy AM. <a href="http://biorxiv.org/content/early/2016/08/24/071282">Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.</a> bioRxiv. (2016).</p>

<p><a href="https://github.com/marbl/canu">Canu download</a></p>

<p><strong>HGAP2</strong></p>

<p>Chin, C.-S., Alexander, D.H., Marks, P., Klammer, A.A., Drake, J., Heiner, C., Clum, A., Copeland, A., Huddleston, J., Eichler, E.E., et al. (2013). <a href="http://www.nature.com/nmeth/journal/v10/n6/abs/nmeth.2474.html">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.</a>  Nat. Methods 10, 563–569.</p>

<p><a href="https://github.com/PacificBiosciences/Bioinformatics-Training/wiki/HGAP-2.0">HGAP2 download</a></p>

<p><strong>Celera Assembler</strong></p>

<p><a href="http://wgs-assembler.sourceforge.net/wiki/index.php?title=Main_Page">Celera download</a></p>

<p><strong>Spades</strong></p>

<p>Nurk, S., Bankevich, A., Antipov, D., Gurevich, A., Korobeynikov, A., Lapidus, A., Prjibelsky, A., Pyshkin, A., Sirotkin, A., Sirotkin, Y., et al. (2013). <a href="http://link.springer.com/chapter/10.1007%2F978-3-642-37195-0_13">Assembling Genomes and Mini-metagenomes from Highly Chimeric Reads.</a> (Springer Berlin Heidelberg), pp. 158–170.  </p>

<p>Antipov, D., Korobeynikov, A., McLean, J.S., and Pevzner, P.A. (2016). <a href="http://bioinformatics.oxfordjournals.org/content/32/7/1009">hybridSPAdes: an algorithm for hybrid assembly of short and long reads.</a> Bioinformatics 32, 1009–1015.</p>

<p><a href="http://bioinf.spbau.ru/spades">Spades download</a></p>

<p><strong>QUAST</strong></p>

<p>Gurevich, A., Saveliev, V., Vyahhi, N., and Tesler, G. (2013). <a href="http://bioinformatics.oxfordjournals.org/content/29/8/1072">QUAST: quality assessment tool for genome assemblies.</a> Bioinformatics 29, 1072–1075.</p>

<p><a href="http://quast.sourceforge.net/">QUAST download</a></p>

<p><strong>MLST</strong></p>

<p><a href="https://github.com/tseemann/mlst">MLST typing</a></p>

<p><strong>Kraken</strong></p>

<p>Wood, D.E., Salzberg, S.L., Venter, C., Remington, K., Heidelberg, J., Halpern, A., Rusch, D., Eisen, J., Wu, D., Paulsen, I., et al. (2014). <a href="https://ccb.jhu.edu/software/kraken/">Kraken: ultrafast metagenomic sequence classification using exact alignments.</a> Genome Biol. 15, R46.
<a href="https://ccb.jhu.edu/software/kraken/">Kraken download</a></p>

<p><strong>Pilon</strong></p>

<p>Walker, B.J., Abeel, T., Shea, T., Priest, M., Abouelliel, A., Sakthikumar, S., Cuomo, C.A., Zeng, Q., Wortman, J., Young, S.K., et al. (2014). <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0112963">Pilon: An Integrated Tool for Comprehensive Microbial Variant Detection and Genome Assembly Improvement.</a> PLoS One 9, e112963.</p>

<p><a href="https://github.com/broadinstitute/pilon/wiki">Pilon download</a></p>

<p><strong>Blobplots</strong></p>

<p>Kumar, S., Jones, M., Koutsovoulos, G., Clarke, M., and Blaxter, M. (2013). <a href="http://dx.doi.org/10.3389/fgene.2013.00237">Blobology: exploring raw genome data for contaminants, symbionts and parasites using taxon-annotated GC-coverage plots.</a> Front. Genet. 4, 237.</p>

<p><a href="https://drl.github.io/blobtools/">Blobtools download</a></p>

<p><strong>SSPACE</strong>
Boetzer, M., Henkel, C. V, Jansen, H.J., Butler, D., and Pirovano, W. (2011). <a href="https://doi.org/10.1186/1471-2105-15-211">Scaffolding pre-assembled contigs using SSPACE.</a> Bioinformatics 27, 578–579.</p>

<p><a href="https://github.com/nsoranzo/sspace_basic">SSPACE download</a></p>

<p><strong>Sequencing data</strong></p>

<p><a href="http://www.ebi.ac.uk/ena/data/view/PRJNA212442"><em>Orientia tsutsugamushi</em> str. Gilliam</a></p>

<p><a href="http://www.ebi.ac.uk/ena/data/view/PRJNA212440"><em>Orientia tsutsugamushi</em> str. Karp</a></p>

<p><a href="http://www.ebi.ac.uk/ena/data/view/PRJNA212441"><em>Orientia tsutsugamushi</em> str. Kato</a></p>

<p><a href="http://www.ebi.ac.uk/ena/data/view/PRJNA212490"><em>Orientia tsutsugamushi</em> str. Sido</a></p>

<p><a href="http://www.ebi.ac.uk/ena/data/view/PRJNA212457"><em>Orientia tsutsugamushi</em> str. TA716</a></p>

<p><a href="http://www.ebi.ac.uk/ena/data/view/PRJNA212454"><em>Orientia tsutsugamushi</em> str. TA763</a></p>

<p><a href="http://www.ebi.ac.uk/ena/data/view/PRJNA232539"><em>Orientia tsutsugamushi</em> str. UT144</a></p>

<p><a href="http://www.ebi.ac.uk/ena/data/view/PRJNA212456"><em>Orientia tsutsugamushi</em> str. UT76</a></p>

<p><a href="http://www.ebi.ac.uk/ena/data/view/PRJNA212443"><em>Orientia chuto</em> str. Fuller</a></p>
      </section>
      <footer>
        <p>This project is maintained by <a href="https://github.com/embatty">embatty</a></p>
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
      </footer>
    </div>
    <script src="javascripts/scale.fix.js"></script>
    
  </body>
</html>
